### ISSUE TITLE
Support super fluid in the new checkout

### ISSUE THREAD

**Issue by searchableguy** [unlock-protocol/unlock#9284]
**Describe the bug**
Superfluid supported landed in classic-checkout recently which we should apply to new-checkout design as well since it doesn't support it currently.

**Comment by julien51** [comment]
Yes, this should be part of this maybe? https://github.com/unlock-protocol/unlock/issues/9201
